### DISCUSSION TITLE
tag NFC

### DISCUSSION THREAD

**Discussion by paolinor88** [indomus/forum#656] [HUB: Home Assistant]
Buongiorno a tutti, ho un problema con la scrittura dei tag NFC. Ho deciso di avventurarmi in questa impresa, ma la frustrazione inizia ad essere elevata...
App per IOS correttamente installata, accesso remoto sicuro (con Nabu Casa) funzionante.. ma quando vado a scrivere il tag mi viene fuori "Stack error". Le ho già provate penso tutte.. il tag funziona, con altre app riesco a scriverlo e leggerlo.
Qualcuno ha avuto questo problema?
Grazie!

**Comment by snakuzzo** [comment]
Troppo poco secondo me per capire  da cosa possa dipendere.
Lato HA logga qualcosa ?

**Reply by paolinor88** [reply]
No, il log non restituisce nulla. Penso perché l'errore (quale che sia) è a monte di HA.
Ti allego uno screen... 
Domanda: nella guida indica come indispensabile il controllo remoto https; io uso HA cloud via Nabu Casa, può esserci un conflitto lì? Non ne vedo il motivo, ma non riuscendo a capire dove diamine stia sbagliando, le provo un po' tutte
![IMG_7292](https://user-images.githubusercontent.com/68273270/131222846-8bb4a259-f764-40ee-91c1-ed30a0596405.PNG)

**Reply by snakuzzo** [reply]
Non credo c'entri qualcosa Nabucasa.
Senza log non saprei che dirti.
Ti consiglio di aprire anche un topic sul [forum ufficiale](https://community.home-assistant.io/) di HA e/o una issue su [Github](https://github.com/home-assistant/core/issues).
Forse prima ancora ti conviene chiedere direttamente sul [canale Discord ufficiale](https://discord.com/channels/330944238910963714/427516175237382144), magari ti risponde qualche sviluppatore

**Reply by paolinor88** [reply]
OK, ci proverò. Se ne vengo a capo posto la soluzione qui.. grazie comunque!

**Comment by paolinor88** [comment]
Ho risolto con i tag del tipo “215”. Per qualche ragione questa tipologia viene letta e scritta correttamente da HA, mentre il modello precedente (che avevo acquistato perché specifico per iPhone) non era scrivibile da HA.
Da tenere a mente! Grazie comunque per l’aiuto
